### PR TITLE
add 2022-04-01 context

### PR DESCRIPTION
**Pull request by Beeter-yong**

### CODE REVIEW

File: meeting_summary/2022-04-01-HTAP概述及存储格式.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Hyper在存储格式的设计首先就是会将一个关系表划分为多个固定大小的Chunk，以一个Chunk为单位对数据在存储格式进行冷热变动调度。刚插入的数据集中在某个chunk数据块中，这时候称该数据还是热数据，因为在最近的时间范围有很大的概率进行读取、修改等操作。但随着时间的推移，这些数据就会成为旧数据，不再经常变更也就是冷数据，就会将对应的 chunk块中的数据转换为列存形式，并为了节省内存而进行压缩处理。
+
+### FSM
+
+首先会对一个关系表按照行进行划分，分成一个个的Tile组，Tile组中对应的元组又可以按列进行划分，分成一个个物理块，也就是一个个Tile，但不同的物理块包含的属性是不相同的，每个物理块总是尽可能将经常一起访问的数据列存储在同一个物理块中。
+
+### Real-Time LSM-Tree for HTAP
+
+LSM-Tree 具备新数据在上层，旧数据在下层特显，以及为了提高读效率而定期 compaction，使得 Lsm-tree 本身就是一个数据生命周期感知敏感的数据布局。基于此，将 LSM 多层结构设计升级未每层数据存储格式可以动态变更，将上层数据向下传递过程中，由行存逐渐列存变换，以支持 HTAP 方案。
```
Comment (ehds) [inline meeting_summary/2022-04-01-HTAP概述及存储格式.md:37]: `特显` should be `特点`
`升级未` should be `升级为`

File: README.md [hunk 2]
```diff
@@ -17,3 +17,4 @@ DSCL weekly group meeting paper list.
 | 2022-02-25 | Database Testing | Database Testing, Optimizer Testing | [doc](./meeting_summary/2022-02-25-数据库系统测试方法介绍.md) |
 | 2022-03-04 | NVM、哈希 | NVM | [doc](./meeting_summary/2022-03-04-面向NVM的数据库哈希索引优化研究.md) |
 | 2022-03-25 | Memory Database | Trie, ART, HOT, Hyperion | [doc](./meeting_summary/2022-03-25-内存数据库简述及索引优化.md) |
+| 2022-04-01 | HTAP、Data formate | HTAP、PAX、Hyper、FSM、RealTime LSM | [doc](./meeting_summary/2022-04-01-HTAP概述及存储格式.md) |
```
Comment (IcePigZDB) [inline README.md:20]:
```suggestion
| 2022-04-01 | HTAP Storage Format | HTAP, PAX, Hyper, FSM, RealTime LSM | [doc](./meeting_summary/2022-04-01-HTAP概述及存储格式.md) |
```

Comment (IcePigZDB) [reply README.md:20]: There is no `、` in English, we'd better use ','.

File: meeting_summary/2022-04-01-HTAP概述及存储格式.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# 2022.04.01 分享纪要
+
+- 分享人:任勇闯
+- 关键词:HTAP概述、存储格式
+- Keywords: HTAP、storage formate
```
Comment (IcePigZDB) [inline meeting_summary/2022-04-01-HTAP概述及存储格式.md:5]: how about keep consistency with README

**Comment by IcePigZDB** [comment]
conflict should be fix

File: README.md [hunk 4]
```diff
@@ -18,3 +18,4 @@ DSCL weekly group meeting paper list.
 | 2022-02-25 | Database Testing | Database Testing, Optimizer Testing | [doc](./meeting-summary/2022-02-25-数据库系统测试方法介绍.md) |
 | 2022-03-04 | NVM for Hashtables | NVM, Hashtables | [doc](./meeting-summary/2022-03-04-面向NVM的数据库哈希索引优化研究.md) |
 | 2022-03-25 | Memory Database | Trie, ART, HOT, Hyperion | [doc](./meeting-summary/2022-03-25-内存数据库简述及索引优化.md) |
+| 2022-04-01 | HTAP、Data formate | HTAP, PAX, Hyper, FSM, RealTime LSM | [doc](./meeting_summary/2022-04-01-HTAP概述及存储格式.md) |
```
Comment (IcePigZDB) [inline README.md:21]:
```suggestion
| 2022-04-01 | HTAP, Data formate | HTAP, PAX, Hyper, FSM, RealTime LSM | [doc](./meeting_summary/2022-04-01-HTAP概述及存储格式.md) |
```

**Review by IcePigZDB — APPROVED** [review]

**Review by ehds — APPROVED** [review]

**Pull request merged**